src/ucp/wireup/select.c

```diff
@@ -769,23 +780,33 @@ static ucs_status_t ucp_wireup_add_am_lane(ucp_ep_h ep, const ucp_ep_params_t *p
     return UCS_OK;
 }
 
-static ucs_status_t ucp_wireup_add_rndv_lane(ucp_ep_h ep,
-                                             const ucp_ep_params_t *params,
-                                             unsigned address_count,
-                                             const ucp_address_entry_t *address_list,
-                                             ucp_wireup_lane_desc_t *lane_descs,
-                                             ucp_lane_index_t *num_lanes_p)
+static ucs_status_t ucp_wireup_add_rndv_lanes(ucp_ep_h ep,
+                                              const ucp_ep_params_t *params,
+                                              unsigned address_count,
+                                              const ucp_address_entry_t *address_list,
+                                              ucp_wireup_lane_desc_t *lane_descs,
+                                              ucp_lane_index_t *num_lanes_p)
 {
+    uint64_t tl_bitmap = -1;
     ucp_wireup_criteria_t criteria;
     ucp_rsc_index_t rsc_index;
     ucs_status_t status;
     unsigned addr_index;
     double score;
+    ucp_address_entry_t *addrs; /* copy of address list */
+    ucp_lane_index_t max_lanes;
+    ucp_lane_index_t i;
 
     if (!(ucp_ep_get_context_features(ep) & UCP_FEATURE_TAG)) {
         return UCS_OK;
     }
 
+    if ((params->field_mask & UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE) &&
+        (params->err_mode == UCP_ERR_HANDLING_MODE_PEER)) {
+        ucs_trace_func("rendezvous: disabled due to error mode");
```

Comment:
minor: we typically use `ucs_trace` for wireup

Reply:
will fix after tests are complete

Reply:
fixed